### PR TITLE
Added non-verbose output for bit depth and color space.

### PR DESCRIPTION
**Pull request by zond**

### CODE REVIEW

File: jxl/src/headers/bit_depth.rs [hunk 1]
```diff
@@ -20,11 +20,11 @@ pub struct BitDepth {
     #[coder_true(u2S(32, 16, 24, Bits(6)+1))]
     #[coder_false(u2S(8, 10, 12, Bits(6)+1))]
     #[default(8)]
-    bits_per_sample: u32,
+    pub bits_per_sample: u32,
```
[the remark this answers is not in the document]
Comment (zond) [reply jxl/src/headers/bit_depth.rs]: Fixed

File: jxl/src/headers/extra_channels.rs [hunk 2]
```diff
@@ -54,7 +54,7 @@ pub struct ExtraChannelInfo {
     // TODO(veluca93): if using Option<bool>, this is None when all_default.
     #[condition(ec_type == ExtraChannel::Alpha)]
     #[default(false)]
-    alpha_associated: bool,
```
Comment (mo271) [inline jxl/src/headers/extra_channels.rs:57]: did you accidentally delete this line or is github just confused?

Comment (zond) [reply jxl/src/headers/extra_channels.rs:57]: I made it `pub`, which GitHub presents as one `-` and one `+`.

File: jxl/src/headers/extra_channels.rs [hunk 3]
```diff
@@ -54,7 +54,7 @@ pub struct ExtraChannelInfo {
...
+    pub alpha_associated: bool,
```
Comment (veluca93) [inline jxl/src/headers/extra_channels.rs:57]: same thing here on pub

Comment (zond) [reply jxl/src/headers/extra_channels.rs:57]: So what's the heuristic for knowing if it should be pub or not. Why is e.g. `dim_shift` pub?

Comment (zond) [reply jxl/src/headers/extra_channels.rs]: Done.

Comment (veluca93) [reply jxl/src/headers/extra_channels.rs]: I'd make dim_shift not pub either nowadays :D

**Review by veluca93 — APPROVED** [review]

**Pull request merged**